pkg/gadgets/trace/tcpretrans/tracer/bpf/tcpretrans.h

```diff
@@ -31,6 +31,7 @@ struct event {
 	__u16 sport;
 	__u8 state;
 	__u8 tcpflags;
+	__u8 type;
```

Comment:
If we make this an enum, it will help later on with https://github.com/inspektor-gadget/inspektor-gadget/issues/1928